tests/emmet-core/qchem/test_qchem.py

```diff
@@ -0,0 +1,126 @@
+import json
+
+import pytest
+from monty.io import zopen
+from monty.serialization import loadfn
+
+from emmet.core.qchem.calc_types import LevelOfTheory, TaskType, level_of_theory, task_type
+from emmet.core.qchem.task import TaskDocument
+from emmet.core.mpid import MPID
+
+
+def test_task_type():
+
+    task_types = [
+        "single-point",
+        "geometry optimization",
+        "frequency analysis",
+        "frequency-flattening geometry optimization",
+        "transition-state geometry optimization",
+        "frequency-flattening transition-state geometry optimization",
+    ]
```

Comment:
I should have caught this earlier, but we do now support "force" jobs, which could end up being important for training NN potentials in the future. I guess we could add that in a later PR though since we've done barely any so far.